nodejs/week2/homework/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -0,0 +1,83 @@
+const express = require("express");
+const router = express.Router();
+
+const meals = require("./../data/meals.json");
+
+router.get("/", async (req, res) => {
+  if(Object.keys(req.query).length === 0 ){
+  res.send(meals);
+} 
+else if
```

Comment:
You should try and format your code better :) 
For instance the curly brackets should be aligned as well as the first if and the else if.
VSCode has a built-in formatter you can use -> have a look at this answer for the shortcuts:
https://stackoverflow.com/a/29973358/7237264